### PR TITLE
Doc migration updates

### PR DESCRIPTION
**Pull request by gallacher**
# Description

Documentation is being moved to the doc portal.  So content is being removed from here and linked to the doc portal/CSM GitHub repository.

# Issues

List the issues impacted by this PR:

| Issue ID |
| -------- |
|          |

# Checklist

- [x] I have performed a self-review of my own code
- [ ] I have run 'make check' to ensure my code does not have any formatting, vetting, linting, or security issues
- [ ] I have run 'make test' to ensure new and existing unit tests pass locally with my changes
- [ ] I have not allowed coverage numbers to degrade
- [ ] I have maintained at least 90% code coverage
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration

- [x] Visual inspection of changes and manual testing of links.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -90,15 +72,16 @@ This will also provide code coverage statistics for the various Go packages.
 
 ## Support
 
-Don’t hesitate to ask! Contact the team and community on [our support page](https://github.com/dell/karavi-observability/blob/main/docs/SUPPORT.md).
-Open an issue if you found a bug on [Github Issues](https://github.com/dell/karavi-observability/issues).
+For all your support needs or to follow the latest ongoing discussions and updates, join our Slack group. Click [Here](http://del.ly/Slack_request) to request your invite.
```
Comment (AronAtDell) [inline README.md:75]: Is the slack link to be filled in?

File: README.md [hunk 2]
```diff
@@ -8,54 +8,36 @@ You may obtain a copy of the License at
...
+# Dell EMC Container Storage Modules (CSM) for Observability - Topology
 
 [![Contributor Covenant](https://img.shields.io/badge/Contributor%20Covenant-v2.0%20adopted-ff69b4.svg)](docs/CODE_OF_CONDUCT.md)
 [![License](https://img.shields.io/github/license/dell/karavi-topology)](LICENSE)
 [![Docker Pulls](https://img.shields.io/docker/pulls/dellemc/karavi-topology)](https://hub.docker.com/r/dellemc/karavi-topology)
 [![Go version](https://img.shields.io/github/go-mod/go-version/dell/karavi-topology)](go.mod)
 [![GitHub release (latest by date including pre-releases)](https://img.shields.io/github/v/release/dell/karavi-topology?include_prereleases&label=latest&style=flat-square)](https://github.com/dell/karavi-topology/releases/latest)
 
-Karavi Topology is part of the Karavi Observability storage enabler, which provides Kubernetes administrators standardized approaches for storage observability in Kuberenetes environments.
+Topology is part of Container Storage Modules (CSM) for Observability, which provides Kubernetes administrators standardized approaches for storage observability in Kuberenetes environments.
```
Comment (AronAtDell) [inline README.md:19]: Should we have Dell EMC prior to "Container Storage Modules" to be consistent?

File: README.md (hunk 1 above)
Comment (gallacher) [reply README.md:75]: We were asked to point everyone to the link above: http://del.ly/Slack_request

**Review by tdawe — APPROVED** [review]

**Review by shaynafinocchiaro — APPROVED** [review]

**Comment by sharmilarama** [comment]
e2e pass

**Pull request merged**